### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Ispirett**
thanks for your time.

### CODE REVIEW

File: scss/_media.scss [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    position: unset !important;
+    opacity: 1 !important;
+    margin: 0 auto !important;
+  }
+  .big-logo-1 {
+    position: unset !important;
+    opacity: 1 !important;
+    margin: 0 auto !important;
+    height: unset !important;
+    margin-top: 5rem !important;
```
Comment (inhaq) [inline scss/_media.scss]: kindly avoid using `!important`

**Review by inhaq — CHANGES_REQUESTED** [review]
Hi Team, good work on making CSS framework and using it as well. I'll recommend making documentation for it as well. You can either update readme or you can also add it to the demo site you created. Kindly avoid using `!important`. Once fixed, resubmit for review.

File: scss/_media.scss (hunk 1 above)
Comment (Ispirett) [reply scss/_media.scss]: Thanks, we fixed it :)

**Review by inhaq — APPROVED** [review]
Great work on writing documentation for the framework. This will definitely very helpful in future open source projects as well.
Keep in mind for future is that `important!` will cause issues when your project size increases and you'll need debugging every time to sort out what's going on. 
Go ahead and submit the project completion form 👍🏻

**Pull request merged**